packages/notifications/src/Notification.php

```diff
@@ -86,6 +103,21 @@ public static function fromArray(array $data): static
         return $static;
     }
 
+    protected function isViewSafe(string $view): bool
+    {
+        return in_array($view, $this->safeViews, strict: true);
+    }
+
+    public function safeViews(string | array $safeViews): static
```

Comment:
@danharrin, only now I see we're accepting string and array here. Since the name of the setter is `safeViews` (plural), wouldn't it make sense to only accept an array...? Would be breaking now I guess, but maybe for v3?

Reply:
I don't mind it as a DX thing TBH

Reply:
Alright then.